fs2/cli/synthesize.py

```diff
@@ -342,10 +342,21 @@ def synthesize(  # noqa: C901
         "--output-type",
         help="""Which format(s) to synthesize to.
         Multiple formats can be provided by repeating `--output-type`.
-        '**wav**' is the default and will synthesize to a playable audio file;
-        '**spec**' will generate predicted Mel spectrograms. Tensors are Mel band-oriented (K, T) where K is equal to the number of Mel bands and T is equal to the number of frames. 
+
+
+        '**wav**' is the default and will synthesize to a playable audio file. Requires --vocoder-path.
+
+
+        '**spec**' will generate predicted Mel spectrograms. Tensors are Mel band-oriented (K, T) where K is equal to the number of Mel bands and T is equal to the number of frames.
+
+
         '**textgrid**' will generate a Praat TextGrid with alignment labels. This can be helpful for evaluation.
+
+
         '**readalong**' will generate a ReadAlong from the given text and synthesized audio (see https://github.com/ReadAlongs).
+
+
+        '**offline-ras**' will generate a single file Offline HTML ReadAlong that can be further edited in the ReadAlong Studio Editor. Also implies '--output-type wav'. Requires --vocoder-path.
```

Comment:
I would prefer to call this `readalong-html` just for continuity with the other one. maybe calling the other one `readalong-xml` or something?

Reply:
Yeah, makes sense. I think I'll go for `readalong-html` and `readalong-xml`.

Reply:
Done